src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -1020,61 +1023,61 @@ private void Model_PropertyChanged(object sender, PropertyChangedEventArgs e)
                     // Do nothing for now
                     break;
                 case nameof(SelectedNumberFormat):
-                    description = Res.DynamoViewSettingMenuNumberFormat;
+                    description = Resources.ResourceManager.GetString("DynamoViewSettingMenuNumberFormat", System.Globalization.CultureInfo.InvariantCulture);
```

Comment:
Is there a way we can use `nameof` so that if we rename the resource string the changes will be captured here?